packages/backend/src/database/migrations/20240108152115_group_attributes.ts

```diff
@@ -0,0 +1,31 @@
+import { Knex } from 'knex';
+
+const userAttributesTableName = 'user_attributes';
+const groupUserAttributes = 'group_user_attributes';
+
+export async function up(knex: Knex): Promise<void> {
+    if (!(await knex.schema.hasTable(groupUserAttributes))) {
+        await knex.schema.createTable(groupUserAttributes, (table) => {
+            table.uuid('group_uuid').notNullable();
+            table.integer('organization_id').notNullable();
+
+            table
+                .foreign(['group_uuid', 'organization_id'])
```

Comment:
we can do this without the extra org_id

Reply:
Ok. Same for line 14 or 19? Do we need those extra constraints?